src/main/python/util.py

```diff
@@ -57,6 +56,55 @@ def config_input_check(text: str, available: int, res: Enum):
             return res.INT_ERROR
 
 
+def build_docker_container(detached=True, container_path='samgomena/deepshare_worker', version='latest', memory=None, cpus=None):
```

Comment:
Minor nit: Rename this to something like `spin_up_container` as this function doesn't build a container.

Reply:
renamed - good idea, `build` is the wrong word here